final/assets/testnet-config

```diff
@@ -0,0 +1,7 @@
+bitcoin-rpcuser=rpcuser
```

Comment:
can we also create a regtest config?

Reply:
can you point me to some sample `regtest` config or will everthing be the same as `testnet`

Reply:
it should be pretty similar.  why don't you draft something and I'll let you know if there are any issues?